Headlines/src/ViewModels/CategoriesContainerViewModel.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  CategoriesContainerViewModel.swift
+//  Canillitapp
+//
+//  Created by Ezequiel Becerra on 24/06/2018.
+//  Copyright © 2018 Ezequiel Becerra. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+import SDWebImage
+
+protocol CategoriesContainerViewModelDelegate: class {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/b49c86a9b05bec3bbe17833b5a716a8293c42cf1/Headlines/src/ViewModels/CategoriesContainerViewModel.swift#L14"></span>Delegate protocols should be class-only so they can be weakly referenced.</td>
    </tr>
  </tbody>
</table>